draft-ietf-ppm-dap.md

```diff
@@ -1845,22 +1845,22 @@ about the client's measurement, violating the privacy goal of DAP. To prevent
 such replay attacks, this specification requires the aggregators to detect and
 filter out replayed reports.
 
-To detect replay attacks, each aggregator keeps track of the set of report IDs
-pertaining to reports that were previously aggregated for a given task. If the
-leader receives a report from a client whose report ID is in this set, it
-either ignores it or aborts the upload sub-protocol as described in
-{{upload-flow}}. A Helper who receives an encrypted input share whose report ID
-is in this set rejects the report as described in
+To detect replay attacks in a given time window, each aggregator is required to
+keep track of the set of reports it has processed for a given task and that have
+timestamps that fall in the time window. Reports that are replayed MUST be
+rejected during the aggreagtion sub-protocol as described in
 {{early-input-share-validation}}.
 
-[OPEN ISSUE: This has the potential to require aggregators to store report ID
-sets indefinitely. See issue#180.]
+Because honest clients choose the report ID at random, it is sufficient to store
+the set of IDs of processed reports. However, implementations may find it
+helpful to track additional information, like the timestamp, so that the storage
+used for anti-replay can be sharded efficiently.
 
 A malicious aggregator may attempt to force a replay by replacing the report ID
 generated by the client with a report ID its peer has not yet seen. To prevent
```

Comment:
I acknowledge that I'm critiquing text that was already present and is not modified by your change but--
```suggestion
A malicious leader may attempt to force a replay by replacing the report ID
generated by the client with a report ID the helper has not yet seen. To prevent
```
Is there any reason not to be explicit about which aggregators we're talking about?

Reply:
I think this is useful if at some point we move to direct-upload architecture.